package.json

```diff
@@ -49,41 +49,43 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "@eslint/eslintrc": "^3.1.0",
-    "@eslint/js": "^9.13.0",
+    "@eslint/eslintrc": "^3.2.0",
+    "@eslint/js": "^9.15.0",
     "@types/diff": "^5.2.3",
     "@types/fs-extra": "^11.0.4",
     "@types/html-minifier-terser": "5.1.1",
     "@types/ini": "^4.1.1",
     "@types/jest": "^29.5.14",
     "@types/json-diff": "^1.0.3",
-    "@types/lodash": "^4.17.12",
-    "@types/node": "^20.17.1",
+    "@types/lodash": "^4.17.13",
+    "@types/node": "^20.17.8",
     "@types/opener": "^1.4.3",
     "@types/readline-sync": "^1.4.8",
     "@types/sax": "^1.2.7",
+    "@types/sql.js": "^1.4.9",
```

Comment:
Yes, I think you're right. And, in fact, I think that perhaps `sql.js` should actually be a _peer dependency_ in FPL since we expose it through part of the API (and allow consumers of the library to pass in their own instance). Either that or we should slightly refactor SQL so that it no longer takes in an instance of sql.js as an argument. The difficulty there is that we can't have an async constructor, so we would likely need to add an `initialize` function that needs to be called after construction.

Reply:
I've updated FPL to remove the sql.js dependency from the API interface (i.e., the constructor arguments). See https://github.com/FHIR/fhir-package-loader/pull/45.

Reply:
I've updated this PR to use the new FPL 2.0.0-beta.2 that no longer exposes sql.js types in the API. See [bf9533b](https://github.com/FHIR/sushi/pull/1543/commits/bf9533be7ac82cae9a978430d77573d404b0010d).